crates/bitwarden/src/client/flags.rs

```diff
@@ -0,0 +1,43 @@
+#[derive(Debug, Default, Clone, serde::Deserialize)]
+pub struct Flags {
+    #[serde(default, rename = "enableCipherKeyEncryption")]
+    pub enable_cipher_key_encryption: bool,
+}
+
+impl Flags {
+    pub fn load_from_map(map: std::collections::HashMap<String, bool>) -> Self {
+        let map = map
+            .into_iter()
+            .map(|(k, v)| (k, serde_json::Value::Bool(v)))
+            .collect();
+        serde_json::from_value(serde_json::Value::Object(map)).expect("Valid map")
```

Comment:
Serde 😢, we could write a proc macro for this but unsure if it's worth the hassle.

Reply:
We could even do a small declarative macro that just does:
```
this_flag = map.get("this_flag").unwrap_or(false);
that_flag = map.get("that_flag").unwrap_or(false);
another_flag = map.get("another_flag").unwrap_or(false);
```

That said I think we can deal with that when we have more flags to handle.